cpp/src/plasma/store.cc

```diff
@@ -571,9 +571,12 @@ void PlasmaStore::disconnect_client(int client_fd) {
 /// be
 /// buffered, and this will be called again when the send buffer has room.
 ///
-/// @param client_fd The client to send the notification to.
-void PlasmaStore::send_notifications(int client_fd) {
-  auto& notifications = pending_notifications_[client_fd].object_notifications;
+/// @param it Iterator that points to the client to send the notification to.
```

Comment:
It's probably worth being even more explicit and saying that we need to do this because `send_notifications` is used in a loop that iterates over `pending_notifications_`.